### DISCUSSION TITLE
Any suggestions for false positive notifications clouds on a car glass thinking its a human.

### DISCUSSION THREAD

**Discussion by paulr1970** [blakeblackshear/frigate#7133] [General]
Any ideas i`ve tried masking out and setting zones which block the glass but its still getting high amount of notifications from this.
Thank you

![image](https://github.com/blakeblackshear/frigate/assets/79212077/a8798b27-01f2-4a1f-bdb1-050a43d9d465)
![image](https://github.com/blakeblackshear/frigate/assets/79212077/f09c7f81-e8da-4a39-aa06-6c44f7959daa)

**Comment by NickM-27** [comment]
Okay a few things:
1. You are using motion masks which do not stop objects from being detected
2. You have large motion masks which are discouraged from being used and will cause problems
3. You do not have `required_zones` set which means the zones are doing nothing to stop these from happening

**Reply by hagak** [reply]
Why are large motion masks discouraged?  What problems do they cause?  I am using some to block trees from triggering a detection in an area I never expect an object to cause motion (except for the trees).

**Reply by NickM-27** [reply]
> I am using some to block trees from triggering a detection in an area I never expect an object to cause motion (except for the trees).

Okay then it's no problem, I could not tell because of the mask and assumed it may have been a street and the mask was being used to block out people / cars

**Reply by hagak** [reply]
Sorry not the OP just wondering if there was an issue with Large masks, understood motion masks do not stop detection only stop the initial trigger to scan the WHOLE image for detection.

**Reply by NickM-27** [reply]
> Sorry not the OP just wondering if there was an issue with Large masks

Large masks that cover areas where an object causes the motion will cause the object to get "stuck" because the motion of that object coming / going is being forcibly ignored

> understood motion masks do not stop detection only stop the initial trigger to scan the WHOLE image for detection.

frigate does not scan the whole image for objects except 1 time when frigate first starts up; motion masks have nothing to do with this.

**Reply by hagak** [reply]
Ok now I am confused.  I thought motion detection event causes Frigate to do an object detection.  Which follows with why a large motion mask would cause an object to get "stuck".  Frigate does object detection across the entire image regadless of motion masks.  Is this not correct?

**Reply by NickM-27** [reply]
> I thought motion detection event causes Frigate to do an object detection.

That is correct

> Frigate does object detection across the entire image regadless of motion masks. Is this not correct?

No, that is wrong. Frigate looks for objects in the region that motion occurred

**Reply by paulr1970** [reply]
Hi
Apologies only just had a chance to look at this. 
Thanks for the reply,  I do have this setup but i still get snapshots (not clips) of people entering outside of the zone areas. Does this look ok ? Following what the docs say.
![image](https://github.com/blakeblackshear/frigate/assets/79212077/b3687a43-f670-4de2-bd64-9fcf280d0b59)

**Reply by NickM-27** [reply]
This config would not work, the indentation is wrong and `motion:` can't be there without anything below it

**Reply by paulr1970** [reply]
Great , thanks Nick. Think i`ve managed to work it out. I havent had any false alerts since. 
I changed it to this, got rid of the motion and fixed the indentation.

![image](https://github.com/blakeblackshear/frigate/assets/79212077/8b8553a4-4a32-428e-af8e-c0c03257e6c4)

**Comment by hagak** [comment]
> > Frigate does object detection across the entire image regadless of motion masks. Is this not correct?
> 
> No, that is wrong. Frigate looks for objects in the region that motion occurred

Oh that I did have incorrectly understood.
